cmd/tracee-ebpf/internal/flags/flags-filter.go

```diff
@@ -208,6 +211,14 @@ func PrepareFilter(filters []string) (tracee.Filter, error) {
 			continue
 		}
 
+		if strings.HasPrefix(filterName, "net") {
```

Comment:
It is nice to add a filter, but if you don't write in the help message that the user needs to write `net=<iface>` no one will know how to use it...